unified-runtime/scripts/core/device.yml

```diff
@@ -835,6 +835,8 @@ decl: static
 ordinal: "0"
 details:
     - "Creates runtime device handle from native driver device handle."
+    - "Will always return the same handle in `phDevice` for a given `hNativeDevice`."
+    - "If `hNativeDevice` is a root-level device, the handle returned in `phDevice` will be one of the handles that can be obtained by calling $xDeviceGet with the $X_DEVICE_TYPE_ALL device type."
```

Comment:
nit: root-level device is OpenCL terminology, is it defined for UR?

Reply:
yeah it's defined in the description of DeviceRetain/DeviceRelease
> If hDevice is a root level device (e.g. obtained with urDeviceGet)